.changeset/rich-dolls-compete.md

```diff
@@ -0,0 +1,34 @@
+---
+"astro": minor
+---
+
+Improves the developer experience of the `500.astro` file.
+
+The special `src/pages/500.astro` page now accepts an error as a prop. It can be anything, so make sure to handle it properly:
```

Comment:
```suggestion
When an error is thrown, the special `src/pages/500.astro` page now automatically receives the error message as a prop. This allows you to display more specific information about the error on a custom 500 page.
```
For the changeset, (vs full complete docs), I wonder if we can get away with not having the second paragraph? I don't like "handle it properly" without getting into an explanation of what they need to worry about/do, and we're writing that in docs anyway. This is just the taste of what's new and cool!

(Also, is the above correct? I like focusing on the use of it)